.eleventy.js

```diff
@@ -150,11 +152,13 @@ module.exports = function(config) {
   // COLLECTIONS
   // ----------------------------------------------------------------------------
   config.addCollection('authors', authors);
+  config.addCollection('newsletters', newsletters);
```

Comment:
it seems weird to add this as a collection because we never use it (IOW we never loop over `collections.newsletters`). I think you can remove this.

Reply:
We use it in `/site/content/en/newsletter/individual.njk` but we can come back to it's use later.